<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTranslatorStatisticsTab</class>
 <widget class="QWidget" name="CTranslatorStatisticsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Time range</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioRangeWeek">
          <property name="text">
           <string>Week</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRangeMonth">
          <property name="text">
           <string>Last month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRangeThisMonth">
          <property name="text">
           <string>This month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRangeYear">
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRangeAll">
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioRangeWeek</tabstop>
  <tabstop>radioRangeMonth</tabstop>
  <tabstop>radioRangeThisMonth</tabstop>
  <tabstop>radioRangeYear</tabstop>
  <tabstop>radioRangeAll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
